server/odc-migrate/src/main/resources/migrate/common/V_4_2_1_6__alter_connect_database.sql

```diff
@@ -0,0 +1,4 @@
+alter table `connect_database` modify `charset_name` varchar(128) default null comment 'database charset name';
```

Comment:
Actually in individual space, we do not need to obtain charset_name, collation_name, and other information, so I remove the NotNull constraint of these columns.